relayer/relays/beefy/parameters.go

```diff
@@ -89,13 +93,37 @@ func toBeefyClientCommitment(c *types.Commitment) *contracts.BeefyClientCommitme
 	}
 }
 
-func cleanSignature(input types.BeefySignature) (uint8, [32]byte, [32]byte) {
+func cleanSignature(input types.BeefySignature) (v uint8, r [32]byte, s [32]byte, err error) {
```

Comment:
Out of interest, why name the return parameters?

Reply:
Nothing special, just for return with error https://github.com/Snowfork/snowbridge/blob/aa41d7d32f0955317f2201336c8b78040d3c76e9/relayer/relays/beefy/parameters.go#L106 easily.